python/python/lancedb/util.py

```diff
@@ -231,6 +222,15 @@ def _(value: str):
     return f"'{value}'"
 
 
+@value_to_sql.register(bytes)
+def _(value: bytes):
```

Comment:
do we want to support more types? like str?

Reply:
? `str` is the function above.

Reply:
string is already supports (see 2 lines above)

Reply:
I meant coercing str to bytes automatically. nvm doesn't seem possible from this interface